modules/polar_obstacle_density.py

```diff
@@ -0,0 +1,85 @@
+"""
+Classes to store object densities as a histogram.
+"""
+
+
+class SectorObstacleDensity:
+    """
+    Class to represent the obstacle density for a specific angular sector.
+    """
+
+    __create_key = object()
+
+    @classmethod
+    def create(
+        cls, angle_start: float, angle_end: float, density: float
+    ) -> "tuple[bool, SectorObstacleDensity | None]":
+        """
+        Create a new SectorObstacleDensity object.
+
+        Parameters:
+        - angle_start: The starting angle of the sector cluster in degrees
+        - angle_end: The ending angle of the sector cluster in degrees
+        - density: The calculated obstacle density for this sector. (0 <= density <= 1)
```

Comment:
How can we interpret these density values? Given a value 0 <= x <= 1, what does x tell me about the obstacles in the sector?